docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -203,19 +219,24 @@ To explore the backups and files held within Restic repository, you must use the
 
 ![Restic Object Storage bucket in Cloud Manager](backup-restic-bucket-object-storage.png)
 
-### Create an alias
+### Create an Alias
 
 It can get tedious typing out the arguments to the Restic command. To make life easier for maintenance and daily management, create an alias for the command with the arguments you need.
 
-In your profile's aliases file, add the line:
+{{< note >}}
```

Comment:
I didn't see an easy way to create this alias for a non-root user. In particular, I couldn't figure out how to use `source` to export the variables in `/root/restic_params` into the shell of the non-root user. So, I ended up just adding a caveat to this section that described how it would only work for `root`

Having said that, I am definitely not an advanced Bash user, so there very well could be a solution here that I didn't think of :)